x-pack/test/security_solution_endpoint/apps/endpoint/resolver.ts

```diff
@@ -207,7 +207,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     });
 
     describe('Resolver Tree events', function () {
-      const expectedData = [
+      let expectedData = [
```

Comment:
I would suggest leaving this as a constant and adding a new constant for the second set of expected data.  Perhaps name them based on the type of event being evaluated.  This will help reduce the risk of someone accidentally adding a test after the initial data has changed, but where it expects the initial data.

Reply:
Done